File: net/Azure.Storage.Blobs.PerfStress/Core/SizeOptions.cs
using Azure.Test.PerfStress;
using CommandLine;

namespace Azure.Storage.Blobs.PerfStress.Core
{
    public class SizeOptions : PerfStressOptions
    {
        [Option('s', "size", Default = 10 * 1024, HelpText = "Size of message (in bytes)")]
        public long Size { get; set; }
    }
}

File: common/perf-test-core/src/main/java/com/azure/perf/test/core/SizeOptions.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

package com.azure.perf.test.core;

import com.beust.jcommander.Parameter;

/**
 * Represents the size option for performance test.
 */
public class SizeOptions extends PerfStressOptions {
    @Parameter(names = { "-s", "--size" }, description = "Size of payload (in bytes)")
    private long size = 10 * 1024;

    /**
     * Get the configured size option for performance test.
      * @return The size.
     */
    public long getSize() {
        return size;
    }
}
